Dockerfile.ocp

```diff
@@ -3,9 +3,15 @@ WORKDIR /go/src/github.com/metal3-io/baremetal-operator
 COPY . .
 RUN make build
 RUN make tools
+COPY /go/src/github.com/metal3-io/baremetal-operator/config/crd/ocp_kustomization.yaml /go/src/github.com/metal3-io/baremetal-operator/config/crd/kustomization.yaml
+RUN make manifests
 
 FROM registry.ci.openshift.org/ocp/4.8:base
 COPY --from=builder /go/src/github.com/metal3-io/baremetal-operator/bin/baremetal-operator /
 COPY --from=builder /go/src/github.com/metal3-io/baremetal-operator/bin/get-hardware-details /
 COPY --from=builder /go/src/github.com/metal3-io/baremetal-operator/bin/make-bm-worker /
 COPY --from=builder /go/src/github.com/metal3-io/baremetal-operator/bin/make-virt-host /
+COPY --from=builder /go/src/github.com/metal3-io/baremetal-operator/config/render/capm3.yaml /manifests/0000_30_baremetal-operator_03_baremetalhost.crd.yaml
```

Comment:
Typo, it's `0000_30_baremetal-operator_01_baremetalhost.crd.yaml`

Reply:
Minor point, but here I'd leave just the copy of the (`/manifests`) subfolder, and have a script in the first part that takes care of the rest; in this way you could also test locally the outcome.

Reply:
Done.